lib/helpers/sassNodeModulesAliasImporter.js

```diff
@@ -0,0 +1,49 @@
+var path = require("path");
+var findNodeModules = require("find-node-modules");
+var fs = require("fs");
+
+var NODE_MODULES_ALIAS = "~";
+var sassFileExtensions = [".sass", ".scss"];
+
+module.exports = function sassNodeModulesAliasImporter(config) {
+	if (typeof config.sassNodeModulesAlias === "string") {
+		NODE_MODULES_ALIAS = config.sassNodeModulesAlias;
+	}
+
+	return function sassNodeModulesAliasImporter(url, prev) {
+		if (url.substr(0, NODE_MODULES_ALIAS.length) === NODE_MODULES_ALIAS) {
+			var i, j, k;
+			var paths = findNodeModules({ cwd: prev, relative: false }).map(function(
+				base
+			) {
+				return path.join(base, url.substr(NODE_MODULES_ALIAS.length));
+			});
+
+			var filepaths = [];
+
+			// Check filepaths with .sass and .scss extensions, also try filenames prefixed with _
+			for (i in paths) {
+				filepaths.push(paths[i]);
+
+				for (j in sassFileExtensions) {
+					var filepath = paths[i] + sassFileExtensions[j];
+					filepaths.push(filepath);
+					var underscoredFilepath = filepath.split("/");
```

Comment:
Will it work on windows which has `\` path separator?

Reply:
Good point